test/services/mapping/goodsshipment/GoodsShipmentBuilderSpec.scala

```diff
@@ -24,32 +24,50 @@ import forms.declaration.DestinationCountriesSpec.correctDestinationCountries
 import forms.declaration.DocumentSpec.correctPreviousDocument
 import forms.declaration.GoodsLocationTestData.correctGoodsLocation
 import forms.declaration.NatureOfTransactionSpec.correctNatureOfTransaction
+import forms.declaration.PreviousDocumentsData
 import forms.declaration.WarehouseIdentificationSpec.correctWarehouseIdentification
 import models.declaration.SupplementaryDeclarationTestData._
 import org.mockito.ArgumentMatchers.{any, refEq}
 import org.mockito.Mockito.verify
 import org.scalatest.mockito.MockitoSugar
 import org.scalatest.{Matchers, WordSpec}
 import services.cache.{ExportItem, ExportsCacheModelBuilder}
+import services.mapping.goodsshipment.consignment.ConsignmentBuilder
 import services.mapping.governmentagencygoodsitem.GovernmentAgencyGoodsItemBuilder
 import wco.datamodel.wco.dec_dms._2.Declaration
 
 class GoodsShipmentBuilderSpec extends WordSpec with Matchers with ExportsCacheModelBuilder with MockitoSugar {
 
-  private val goodsShipmentNatureOfTransactionBuilder = mock[GoodsShipmentNatureOfTransactionBuilder]
-  private val consigneeBuilder = mock[ConsigneeBuilder]
+  private val mockGoodsShipmentNatureOfTransactionBuilder = mock[GoodsShipmentNatureOfTransactionBuilder]
```

Comment:
minor again but `mock` just adds clutter to the variable names

Reply:
don think this matters...its a mock... call it a mock